### PR TITLE
added info regarding the expectations for images/files

### PR DESCRIPTION
**Pull request by morganen**

### CODE REVIEW

File: content/md/productivity/get-started-with-the-rules-engine.md [hunk 1]
```diff
@@ -917,7 +917,7 @@ value: ../../../
  src/PimEnterprise/Bundle/InstallerBundle/Resources/fixtures/icecat_demo/images/AKNTS_PB.jpg
 ```
 
-We expect a **text** as `value`.
+We expect a **text** as `value` (it corresponds to the filename).
```
Comment (nmarniesse) [inline content/md/productivity/get-started-with-the-rules-engine.md:920]: `corresponds` seems weird to me, event if this word exists in english. I would use `it should match the filename`.  
Maybe ask to another person :smile:

Comment (morganen) [reply content/md/productivity/get-started-with-the-rules-engine.md:920]: @camillebourjade what do you think about "corresponds"? Thanks :)

Comment (camillebourjade) [reply content/md/productivity/get-started-with-the-rules-engine.md:920]: I agree with @nmarniesse :-) 
The 'expect' should  be 'require'

**Review by camillebourjade — APPROVED** [review]
English Fix

**Pull request merged**